src/utils.js

```diff
@@ -29,12 +29,19 @@ const updateUrl = (data) => {
     })}`);
   }
 };
+
 const removeTrailingSlash = path => path.replace(/\/$/, '');
 
+const hasFeatureFlagEnabled = (featureFlag) => {
```

Comment:
utils feels like a dumping ground and I don't think you'd want others to use this outside of config. I would recommend keeping this private `config/index.js`